src/validate.ts

```diff
@@ -88,6 +88,7 @@ function validate(value: any, schema: CompiledDefinition): ValidationError | und
 }
 
 export function request(compiledPath: CompiledPath | undefined,
+                        pathParameters: { [name: string]: any },
```

Comment:
This should go at the end of the parameter list, and be optional, to make sure we don't break existing code.